egs/aishell2/asr1/RESULT.md

```diff
@@ -0,0 +1,33 @@
+# Transformer result (default transformer with initial learning rate = 1.0 and epochs = 25)
```

Comment:
Sorry, I missed these messages because I forgot to set a notification.
1. I received the Google drive link and upload the model.
2. The results (9.1%| 7.7%| 8.7%) are good than TDNN-Chain model (9.59%| 8.81%| 10.87%), but a little worse than another open-source Transformer implement as in https://github.com/shiyuzh2007/ASR, (8.16%| 7.64%| 8.24%).
3. The recent SOTA results on Aishell 2 corpus maybe this paper: https://arxiv.org/abs/1905.11235v2, whose results are (6.20%| 5.86%| 6.40%). It's not a Transformer framework.